docs/sources/datasources/elasticsearch/query-editor/index.md

```diff
@@ -15,64 +15,144 @@ labels:
 
-## Name a time series
+- **@timestamp** - Depicts timestamp data options. Set this when configuring the Elasticsearch data source in the **Time field name** under the [**Elasticsearch details**](/docs/grafana/latest/datasources/elasticsearch/configure-elasticsearch-data-source/#elasticsearch-details) section.
 
-You can control the name for time series via the `Alias` input field.
+- **Interval** - Group by a type of interval. There are option to choose from the dropdown menu to select seconds, minutes, hours or day. You can also add a custom interval such as `30d` (30 days). `Auto` is the default option.
 
-| Pattern              | Replacement value                      |
-| -------------------- | -------------------------------------- |
-| `{{term fieldname}}` | Value of a term group-by               |
-| `{{metric}}`         | Metric name, such as Average, Min, Max |
-| `{{field}}`          | Metric field name                      |
+  - **Min doc count** - The minimum amount of data to include in your query. The default is `0`.
+  - **Thin edges** - Select to trim edges on the time series data points. The default is `0`.
+  - **Offset** - Changes the start value of each bucket by the specified positive(+) or negative (-) offset duration. Examples include `1h` for 1 hour, `5s` for 5 seconds or `1d` for 1 day.
+  - **Timezone** - Select a timezone from the dropdown menu. The default is `Coordinated universal time`.
 
-## Control pipeline metrics visibility
+Click the **+ sign** to add multiple group by options. The data will grouped in order (first by, then by).
 
-Some metric aggregations, such as _Moving Average_ and _Derivative_, are called **Pipeline** aggregations.
-Elasticsearch pipeline metrics must be based on another metric.
-Use the eye icon next to the metric to prevent metrics from appearing in the graph.
-This is useful for metrics you only have in the query for use in a pipeline metric.
+{{< figure src="/static/img/docs/elasticsearch/group-by-then-by-10.2.png" max-width="850px" class="docs-image--no-shadow" caption="Group by options" >}}
 
-{{< figure src="/static/img/docs/elasticsearch/pipeline-aggregation-editor-7-4.png" max-width="500px" class="docs-image--no-shadow" caption="Pipeline aggregation editor" >}}
+### Logs query type
 
-## Create a query
+Logs queries analyze Elasticsearch log data. You can configure the following options:
 
-Write the query using a custom JSON string, with the field mapped as a [keyword](https://www.elastic.co/guide/en/elasticsearch/reference/current/keyword.html#keyword) in the Elasticsearch index mapping.
+- **Logs Options/Limit** - Limits the number of logs to analyze. The default is `500`.
 
-If the query is [multi-field](https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-fields.html) with both a `text` and `keyword` type, use `"field":"fieldname.keyword"` (sometimes `fieldname.raw`) to specify the keyword field in your query.
+#### Logs volume panel
```

Comment:
I would mention that Logs volume panel and Logs panel are available in Explore. 

In dashboard, when you run logs query, you need to select logs panel as visualisation. Moreover, in dashboard, you can't select logs volume panel.